bin/Cores/cores.json

```diff
@@ -515,6 +515,11 @@
     "name": "PokeMini",
     "systems": [24]
   },
+  "potator_libretro":{
+    "name": "Potator",
+    "extensions": "sv",
```

Comment:
May as well remove this line. While sv is the primary extension, bin is supported as well, and removing the extension line entirely allows all extensions the core supports.